lib/rubygems/test_case.rb

```diff
@@ -675,7 +675,7 @@ def util_remove_gem(spec)
   # Removes all installed gems from +@gemhome+.
 
   def util_clear_gems
-    FileUtils.rm_rf File.join(@gemhome, "gems") # TODO: use Gem::Dirs
```

Comment:
Isn't `@gemhome` same as `Gem.paths.home` now? Maybe that was the original idea.

Reply:
Maybe 🤔 will check

Reply:
They are not @simi